wiki/en/Software-Manual.md

```diff
@@ -200,7 +200,7 @@ connection properties. Three buffer sizes are supported:
 Some sound card drivers do not allow the buffer delay to be changed from within the Jamulus software.
 In this case the buffer delay setting is disabled and has to be changed using the sound card driver. On Windows, press the ASIO Setup button to open the driver settings panel.
 
-On Linux, use the JACK configuration tool to change the buffer size.
+On Linux, use the JACK configuration tool to change the buffer size or, if using PipeWire's JACK server, change PipeWire's Quantum parameter with its own configuration tools.
```

Comment:
```suggestion
On Linux:
- when using `jackd`, use the `QJackCtl` to change the buffer size;
- when using PipeWire's JACK server, change PipeWire's Quantum parameter with its own configuration tools.
```

Reply:
The same here. Why jackd?
Second thing, you can use another tool to change the buffer size. Why do we have to talk about QjackCtl?

Reply:
At least be consistent in how things are described.

Reply:
Ok thanks.